### PR TITLE
SISRP-44520 - Terms not appearing in Student Success card for Advisor

### PR DESCRIPTION
**Pull request by nkutub**
https://jira.berkeley.edu/browse/SISRP-44520

### CODE REVIEW

**Comment by davidlimcheng** [comment]
@nkutub, looking at the JIRA comments, it seems like there might have been design considerations for the issue.  Was there more communication done offline for this solution?

File: app/models/student_success/term_gpa.rb [hunk 1]
```diff
@@ -9,7 +9,13 @@ def initialize(opts={})
     end
 
     def merge(data={})
-      data[:termGpa] = get_term_gpas
+      term_gpas = get_term_gpas
+      data[:termGpaWithZero] = get_term_gpas.clone
```
Comment (davidlimcheng) [inline app/models/student_success/term_gpa.rb]: @nkutub I think this should be `term_gpas.clone`

File: app/models/student_success/term_gpa.rb [hunk 2]
```diff
@@ -9,7 +9,13 @@ def initialize(opts={})
...
+      data[:termGpa] = remove_zero_gpas(term_gpas)
```
Comment (davidlimcheng) [inline app/models/student_success/term_gpa.rb]: If this is no longer being used anywhere, can we just get rid of it?

Comment (nkutub) [reply app/models/student_success/term_gpa.rb]: @davidlimcheng I don't understand ? both are being used

File: app/models/student_success/term_gpa.rb (hunk 1 above)
Comment (nkutub) [reply app/models/student_success/term_gpa.rb]: good catch

**Comment by nkutub** [comment]
> @nkutub, looking at the JIRA comments, it seems like there might have been design considerations for the issue. Was there more communication done offline for this solution?

@davidlimcheng not sure what you are referring to

File: app/models/student_success/term_gpa.rb (hunk 2 above)
Comment (davidlimcheng) [reply app/models/student_success/term_gpa.rb]: I see now that it is still being used in `chartGpaTrend()`.

**Comment by davidlimcheng** [comment]
@nkutub I'm just asking if for clarification on the fix for this.  Seems like there was an open-ended discussion on the JIRA ticket, but you must have communicated with the designer/BSA outside of the JIRA ticket.

**Pull request merged**